packages/traceloop-sdk/src/lib/tracing/index.ts

```diff
@@ -301,6 +329,60 @@ export const startTracing = (options: InitializeOptions) => {
   });
 
   _sdk.start();
+
```

Comment:
I think it's a leftover from when you merged to `main`?

Reply:
Shall I remove it?

Reply:
Ofc, unless you meant to add it?

Reply:
I see it still exists in the main branch.